umi/lumi/tests/conftest.py

```diff
@@ -18,8 +18,12 @@ def build_dir(pytestconfig):
 
 @pytest.fixture
 def lumi_dut(build_dir, request):
+
     dut = SbDut('testbench', default_main=True, trace=False)
 
+    # dut = SbDut('testbench', cmdline=True,
+    #             default_main=True, trace=True, trace_type='vcd')
+
```

Comment:
any values in keeping this around?

Reply:
Just as a reminder for the next person...